_includes/default-header.html

```diff
@@ -1,30 +1,41 @@
-<nav class="navbar is-fixed-top" role="navigation" aria-label="main navigation">
-  <div class="container">
-    <div class="navbar-brand">
-      
-      <a class="navbar-item" href="{{ site.baseurl }}/">
-          <img src="{{ site.baseurl }}/images/logo.png" alt="DNAnalyzer">
+<nav class="navbar" role="navigation" aria-label="main navigation">
+  <div class="navbar-brand">
+      <a class="navbar-item logo" href="{{ site.baseurl }}/">
+          DNAnalyzer
```

Comment:
We could maybe have the logo there
@teresa-m what do you think?

Reply:
I think we do not have the logo as svg yet. It is taken from here: https://github.com/StreetScienceCommunity/DNAnalyzer/blob/main/images/Game_banner.svg
@YedilSerzhan Do you want to change the colors of the logo so it fits the once of the website?